scripts/import_templates_gen3.py

```diff
@@ -87,8 +87,9 @@ def _export(butler, export_file):
     runs : iterable [`str`]
         The names of the runs containing exported templates.
     """
-    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=butler.collections)
-    skymap_query = f"skymap IN ({', '.join(repr(id['skymap']) for id in skymaps)})"
+    skymaps = butler.registry.queryDataIds("skymap", datasets=TEMPLATE_NAME, collections=TEMPLATE_COLLECT)
```

Comment:
Why the change from `butler.collections` to `TEMPLATE_COLLECT`? I wouldn't expect `templates/goodSeeing` to exist in the source repository, in general.

Reply:
Thank you for catching that! This was a mistake while trying to get the HSC version of this script to work. I was on the wrong tab, and I guess I didn't undo enough times.